lib/datadog/core/configuration/settings.rb

```diff
@@ -475,6 +475,15 @@ def initialize(*_)
             o.default { env_to_bool(Core::Telemetry::Ext::ENV_ENABLED, true) }
             o.lazy
           end
+
+          # The interval in seconds when telemetry must be sent.
+          #
+          # @default `DD_TELEMETRY_HEARTBEAT_INTERVAL` environment variable, otherwise `60`.
+          # @return [Float]
+          option :heartbeat_interval do |o|
```

Comment:
This comment is not a blocker but more of an open question. 

Should we try to standardize the name of our configuration options when there are similar options for other components? 

For example the remote component has an interval as well, but the configuration is `poll-interval_seconds` https://github.com/DataDog/dd-trace-rb/blob/6c3b37b5195dcfd39c17a60125230d92a72b6223/lib/datadog/core/configuration/settings.rb#L496

I think would be good to have a more consistent configurations name.

cc @lloeki

Reply:
We've been trying to keep configuration methods close to environment variable names, which in this case is `DD_TELEMETRY_HEARTBEAT_INTERVAL`.

`DD_TELEMETRY_HEARTBEAT_INTERVAL` is implemented across multiple languages already, so we can't change the env var.

Also in this case, the heartbeat is not pulling, it's pushing data to the agent.

I'm not against the suggestion, and agree we should standardize these. We've been trying to standardize them so they look like the env vars so far.

Reply:
+1 on including units on things that don't have more specific types (e.g. seconds if it's just a "naked" integer). 

I agree with Marco that the public naming of this is harder to go back and fix, but I would suggest that elsewhere internally in our code we could use `heartbeat_interval_seconds`.